### PR TITLE
{tools}[GCC/8.3.0] byobu v5.133

### PR DESCRIPTION
**Pull request by jordiblasco**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/b/byobu/byobu-5.133-GCC-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+description = """Byobu is an elegant enhancement of the otherwise functional, plain, practical GNU Screen. Byobu
+includes an enhanced profile, configuration utilities, and system status notifications for the GNU screen window
+manager as well as the Tmux terminal multiplexer"""
+
+toolchain = {'name': 'GCC', 'version': '8.3.0'}
+
+builddependencies = [
+    ('Autoconf', '2.69', '', ('GCCcore', '8.3.0')),
+    ('Automake', '1.16.1', '', ('GCCcore', '8.3.0')),
+    ('Autotools', '20180311', '', ('GCCcore', '8.3.0')),
```
Comment (boegel) [inline easybuild/easyconfigs/b/byobu/byobu-5.133-GCC-8.3.0.eb:23]: `Autoconf` and `Automake` are part of the `Autotools` bundle, so no need to include those, and you don't have to hardcode the toolchain either, so please change to:

```python
builddependencies = [
    ('Autotools', '20180311'),
]

File: easybuild/easyconfigs/b/byobu/byobu-5.133-GCC-8.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+]
+
+osdependencies = [('screen', 'tmux')]
+
+
```
Comment (boegel) [inline easybuild/easyconfigs/b/byobu/byobu-5.133-GCC-8.3.0.eb:28]: please remove duplicate empty line

File: easybuild/easyconfigs/b/byobu/byobu-5.133-GCC-8.3.0.eb [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+source_urls = ['https://launchpad.net/%(name)s/trunk/%(version)s/+download/']
+sources = ['%(name)s_%(version)s.orig.tar.gz']
+checksums = ['4d8ea48f8c059e56f7174df89b04a08c32286bae5a21562c5c6f61be6dab7563']
```
Comment (boegel) [inline easybuild/easyconfigs/b/byobu/byobu-5.133-GCC-8.3.0.eb:31]: This block is usually *above* the `(build)dependencies`, please change.

File: easybuild/easyconfigs/b/byobu/byobu-5.133-GCC-8.3.0.eb [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+sanity_check_paths = {
+    'files': ['bin/%(name)s'],
```
Comment (boegel) [inline easybuild/easyconfigs/b/byobu/byobu-5.133-GCC-8.3.0.eb:34]: Does it make sense to also include a sanity check command like `byobu --help`?

**Review by jordiblasco — COMMENTED** [review]
done

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3501.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/90a8ed7818c7dc28a28d9eed322622dd for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3155.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/c65df8f8f357fae39d4f1037f3301c77 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Going in, thanks @jordiblasco!

**Pull request merged**